app/assets/javascripts/discourse/routes/preferences-second-factor.js.es6

```diff
@@ -0,0 +1,21 @@
+import RestrictedUserRoute from "discourse/routes/restricted-user";
+
+export default RestrictedUserRoute.extend({
+  model: function() {
```

Comment:
instead of `model: function()` you should use the new syntax of `model()`, ditto for the other functions in this class.